fem/tmop_tools.cpp

```diff
@@ -352,9 +368,10 @@ void InterpolatorFP::SetInitialField(const Vector &init_nodes,
 }
 
 void InterpolatorFP::ComputeAtNewPosition(const Vector &new_nodes,
-                                          Vector &new_field)
+                                          Vector &new_field,
+                                          int new_nodes_ordering)
 {
-   finder->Interpolate(new_nodes, field0_gf, new_field);
+   finder->Interpolate(new_nodes, field0_gf, new_field, new_nodes_ordering);
```

Comment:
Can't you get the `ordering` from `field0_gf`? Then you won't have to propagate the ordering parameter through the chain of function calls.

Reply:
The `new_nodes_ordering` corresponds to the ordering of the points to be found, which we can't get from `new_nodes` because `new_nodes` is of type `Vector`.